sql/core/src/test/resources/sql-tests/results/subquery/in-subquery/not-in-unit-tests-multi-column-literal.sql.out

```diff
@@ -16,8 +16,7 @@ struct<>
 
 
 -- !query 1
--- Case 5
-  -- (one null column with no match -> row is returned)
+-- (one null column with no match -> row is returned)
```

Comment:
Also, can we remove this too?

Reply:
This will be removed by https://github.com/apache/spark/pull/25357#discussion_r311405432